### ISSUE TITLE
Course packet issues

### ISSUE THREAD

**Issue by jzthompson** [maltyeva/mindimp-pg#76]
1. Currently I can only add one document at a time to a course packet. If I want to add ten documents, I have to create the course packet with one document, and then edit the course packet nine times and upload one document at a time during each edit. Could an option to "add another document" be added to the "create course packet" and "edit course packet" screens?

2. Sometimes when I try to upload a document I get sent to a screen that says "application error". If I keep hitting refresh when I'm on this page the document eventually uploads and all is well. Would be nice to get rid of this error though.

3. When I edit a course packet, I can see the document titles that are already uploaded to the course packet, but there are options to upload new files to each of them. This was initially confusing to me, as I thought I would have to re-upload everything on each edit. I don't; the files are still there, but it would be nice if the edit screen showed that files had already been uploaded for these titles. The picture should hopefully show an example of what I mean.
![capture](https://cloud.githubusercontent.com/assets/10520630/6941314/ce0aa9d6-d84c-11e4-9cb6-a5b4ca53a193.PNG)

4. I found that entering a description for the file I upload is essential. If I don't include a subscription, no download link for the file appears in the course packet. It would be good if the file description were marked as required, or if something were changed so that files could be uploaded without descriptions.

**Comment by maltyeva** [comment]
Hey Jeff, 

I'm still trying to figure out the .rar upload issue, I think it's something with S3. Would it be possible to compress the attachments as .zip in the meantime? I just tested that it works in production. 

I added the description field and will work on the other things today. To allow multiple uploads at once, I'll need to add flash to the page, which may take a little bit.

**Comment by jzthompson** [comment]
Yeah, that's fine Maria. The .rar issue is not critical. I think the only
class it might be pertinent for is art history, and we won't be offering
that course until at least May. Anyway, I can upload files with .zip or.
jpg, so don't worry about .rar if it's too much of a hassle.
